Bottom: 0b2742fde5ea2dd2da792a9f70eaa9249484de00
Top:    cb008219524a6d94f39ef584efae688ed3644114
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-13 14:57:41 +0000

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 9b2f76e..296eeeb 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
